tests/acceptance/pageObjects/FilesPageElement/publicLinksDialog.js

```diff
@@ -143,9 +143,14 @@ module.exports = {
         locateStrategy: this.elements.publicLinkDeleteButton.locateStrategy,
         selector: linkRowDeleteButtonSelector
       }
+      const deleteModalConfirmButton = {
+        selector: this.elements.publicLinkDeletionModalConfirmButton
+      }
       return this.waitForElementVisible(linkRowDeleteButton)
         .initAjaxCounters()
         .click(linkRowDeleteButton)
+        .waitForElementVisible(deleteModalConfirmButton)
```

Comment:
```suggestion
        .waitForElementVisible(publicLinkDeletionModalConfirmButton)
```

Reply:
I need to revert those, the `deleteModalConfirmButton` is a local var inside the function (tried following the existing test structure), while `publicLinkDeletionModalConfirmButton` is part of the selectors and can only be called via `this.elements.publicLinkDeletionModalConfirmButton`

Reply:
whoops, sorry